src/app/item-actions/ItemActionsDropdown.tsx

```diff
@@ -61,16 +62,16 @@ export default React.memo(function ItemActionsDropdown({
     )
   );
 
-  const bulkTag = loadingTracker.trackPromise(async (selectedTag: TagValue) => {
+  const bulkTag = loadingTracker.trackPromise(async (selectedTag: TagCommand) => {
     // Bulk tagging
     const tagItems = filteredItems.filter((i) => i.taggable);
 
-    if (isTagValue(selectedTag)) {
+    if (isTagCommand(selectedTag)) {
```

Comment:
I think this is redundant now with the separate types for tagging and locking? Or maybe I'm missing something.

Also getting some typescript-eslint warnings about now unneeded `as TagValue` type assertions in `dim-api/reducer.ts`.

Reply:
`TagCommand` includes the `clear` value that isn't in `TagValue`. Let me check out the eslint warnings.